src/declarations/backend/backend.did

```diff
@@ -65,6 +65,8 @@ service : (Arg) -> {
   personal_sign : (text) -> (text);
   remove_user_custom_token : (UserTokenId) -> ();
   remove_user_token : (TokenId) -> ();
+  set_many_user_custom_tokens : (vec UserToken) -> ();
+  set_user_custom_token : (UserToken) -> ();
```

Comment:
Mostly because existing ERC20 endpoints are defined as `add_user_token` and `remove_user_token` and we discussed last week that in the future we migrate those to the new tokens feature we are introduced, therefore felt like using a naming pattern that was close - e.g. `add_user_token` -> ``add_user_custom_ token`.

Furthermore, the name `Token` is already used for the struct of the ERC20 token. So to spare a refactoring, I picked "custom" as prefix.

Not against renaming ERC20 related existing code and using other terms for the new code.

We need one struct that contains the token, enabled flag and timestamp and a struct that contains the effective token (ICRC or something else).

Any suggestions?